movie-app/src/components/MoodGallery/MoodContent.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { Box } from '@mui/material';
+import { BASE_IMG_URL, isValid } from '../../data';
+import useFetch from '../../hooks/useFetch';
+import MovieModal from '../MovieModal';
+
+export default function MoodContent({ genre, setTotalPages, currentPage }) {
+  const { data, isLoading, error } = useFetch(
+    `/api/fetch-movie-mood?moodId=${genre}&pageId=${currentPage}`,
+  );
+  const { total_pages: totalPages } = data;
+  const { results: movies } = data || {};
+
+  useEffect(() => {
+    totalPages && setTotalPages(totalPages);
+  }, [setTotalPages, totalPages]);
+
+  const [selectedMovie, setSelectedMovie] = useState({});
+  const [open, setOpen] = useState(false);
+  const handleOpen = (movieData) => {
+    setOpen(true);
+    setSelectedMovie(movieData);
+  };
+  const handleClose = () => setOpen(false);
+
+  return (
+    <>
+      {isLoading ? (
+        <p>Loading...</p>
```

Comment:
I think this is the reason for the increase in the height of the tab panel.
How about using Position Absolute?

Reply:
Ohhh I think you are right!! Thank you thank you thank you 🙏